aspnetcore/signalr/hubcontext.md

```diff
@@ -0,0 +1,56 @@
+---
+title: SignalR HubContext
+author: rachelappel
+description: Overview of the use of SignalR's HubContext
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 6/12/18
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/HubContext
+---
+
+# Sending Messages Outside of you Hub Class
+
+By [Mikael Mengistu](https://github.com/mikaelm12)
+
+The SignalR Hub is the core abstraction for sending messages to clients connectied to our SignalR server. You aren't limited to sending messages to clients from within a Hub though. Here we'll look at how to access a SignalR HubContext to send notification to clients from outside of a hub.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/signalr/hubcontext/sample/) [(how to download)](xref:tutorials/index#how-to-download-a-sample)
+
+## Getting and Using the HubContext
+
+In ASP.NET Core SignalR you can get access to an instance of HubContext via dependency injection. You can get an instance of `HubContext<YourHubType>` in an MVC Controller or middleware this way. In previous versions of SignalR you would do this by explicitly asking for an instance of HubContext via a ConnectionManager but that has been replaced in favor of our dependency injection approach.
+
+Getting an instance of HubContext in a Controller
```

Comment:
Suggest: "Inject an instance of `IHubContext` into a Controller by adding it to your constructor:"

Reply:
Or, make this an h3 (`###`) and put the sentence I suggested below it. Though I think "Getting" still needs to go because gerunds (`-ing` verbs) aren't generally allowed in our docs. I like "Inject" as a replacement verb here.